app/Authentication/AuthServiceProvider.php

```diff
@@ -160,8 +160,13 @@ function () use ($container): ?ApiKey {
         $container->addShared(
             ConsumerReadRepository::class,
             function () use ($container): ConsumerReadRepository {
-                return new InMemoryConsumerRepository(
-                    new CultureFeedConsumerReadRepository($container->get('culturefeed'), true)
+                return new CachedConsumerReadRepository(
+                    new CultureFeedConsumerReadRepository($container->get('culturefeed'), true),
+                    CacheFactory::create(
+                        $container->get(RedisClient::class),
+                        'consumer_repository',
```

Comment:
Nit: not sure if `consumer_repository` is a good namespace, should it be something like `culturefeed_consumer`